invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/users/search/SearchResultItemLayout.js

```diff
@@ -45,18 +45,27 @@ class SearchResultItemComponent extends Component {
         <Table.Cell key={`user-name-${result.id}`} data-label={i18next.t("Username")}>
           @{result.username}
           {result.identities.orcid && (
-            <img
-              className="inline-id-icon ml-5"
-              src="/static/images/orcid.svg"
-              alt="ORCID"
-            />
+            <a href={`https://orcid.org/${result.identities.orcid}`} rel="noreferrer">
+              <img
+                className="inline-id-icon ml-5"
+                src="/static/images/orcid.svg"
+                alt="ORCID"
+              />
+            </a>
+          )}
+          {result.identities.github && (
+            <a
+              href={`https://api.github.com/user/${result.identities.github}`}
+              rel="noreferrer"
```

Comment:
can you fix the `rel` as commented in the other PR?
It all depends if you want to open in a new tab or not

Reply:
Personally I don't think we have a [good excuse](https://css-tricks.com/use-target_blank/) to open in a new tab, I'll remove the noreferrer as we trust GitHub